container-engine-lib/lib/backend_impls/docker/docker_kurtosis_backend/engine_functions/github_auth_storage_creator/github_auth_storage_creator.go

```diff
@@ -95,6 +96,15 @@ func (creator *GitHubAuthStorageCreator) CreateGitHubAuthStorage(
 	return nil
 }
 
+// GetBestEffortGitHubAuthToken Returns empty string if no token found in [githubAuthTokenFile] or [githubAuthTokenFile] doesn't exist
+func GetBestEffortGitHubAuthToken() string {
```

Comment:
`s/GetBestEffort/Get`, I think if it's documented in the function spec/comment that behavior is empty string when no token, we don't need to add `BestEffort`